src/main/cpp/DragonVision/DragonCamera.h

```diff
@@ -23,22 +23,22 @@
 #include "frc/geometry/Pose3d.h"
 #include "frc/DriverStation.h"
 #include "DragonVision/DragonVisionStructs.h"
+#include "DragonVision/DragonVision.h"
 
 class DragonCamera
 {
 public:
-    enum PIPELINE
+    enum VISION_ALIGNMENT
     {
-        OFF,
-        UNKNOWN,
-        MACHINE_LEARNING,
-        APRIL_TAG,
-        COLOR_THRESHOLD
+        NOTE = DragonVision::VISION_ELEMENT::NOTE,
+        SPEAKER = DragonVision::VISION_ELEMENT::SPEAKER,
+        UNKNOWN = 0,
+        APRIL_TAG
     };
 
     DragonCamera(
         std::string cameraName,                /// <I> camera name/type
-        PIPELINE pipeline,                     /// <I> enum for pipeline
+        VISION_ALIGNMENT visionAlignment,      /// <I> enum for pipeline
```

Comment:
Don't change the signature for DragonCamera, we'll cast the enum values when calling vision stuff